app/controllers/search_module.js

```diff
@@ -90,6 +99,10 @@ export default Ember.Controller.extend(InfinityRoute, {
     this.set("filteredResults", []);
   },
 
+  stringifyArray(stateOrType) {
+    return this.get("utilityMethods").arrayExists(stateOrType) ? stateOrType.toString() : '';
```

Comment:
I think using `Array.isArray(stateOrType)` would be better choice. What do you think?